packages/at_chops/example/at_chops_example.dart

```diff
@@ -0,0 +1,4 @@
+import 'package:at_chops/at_chops.dart';
```

Comment:
Are you planning to write an example of usage?

Reply:
yes Gary..in the comments in mentioned few pending items such as readme, example code

Reply:
done